index.js

```diff
@@ -36,6 +36,10 @@ app.get('/mentions-legales', (req, res) => {
   res.render('legalNotice')
 })
 
+app.get('/suivi', (req, res) => {
+  res.render('suivi')
+})
+
 module.exports = app.listen(port, () => {
   console.log(`${appName} listening at http://localhost:${port}`)
 })
```

Comment:
juste petit commentaire comme ça, j'ai pour habitude de toujours avoir une ligne vide en fin de fichier.

Explication : https://thoughtbot.com/blog/no-newline-at-end-of-file 

Il y a beaucoup d'article sur le sujet :)

Reply:
merci, j'avais aucune idée d'où venait du pourquoi de ces messages